mod_reforged/hooks/items/weapons/oriental/nomad_sling.nut

```diff
@@ -4,5 +4,6 @@
 	{
 		create();
 		this.m.Reach = 0;
+		this.m.WeaponType = ::Const.Items.WeaponType.Sling;
```

Comment:
Use `addWeaponType` instead of setting the weapon type. Because the `WeaponType` is used for many things including weapon mastery, so for now we want to keep the `Throwing` weapon type.

Reply:
I wanted to use this opportunity to fix Slings by replacing throwing with slings.

That second bit at the end was actually the important argumentation. If you want to keep that weird other weapon type for now I can see that.